holidays/countries/sint_maarten.py

```
@@ -0,0 +1,118 @@
+#  holidays
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)